src/csharp/Grpc.Tools.MSBuildIntegration.nuspec

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package>
+  <metadata>
+    <id>Grpc.Tools.MSBuildIntegration</id>
+    <title>gRPC C# MSBuild based Tools</title>
+    <summary>MSBuild based tools for C# implementation of gRPC - an RPC library and framework</summary>
+    <description>Precompiled protobuf compiler, gRPC protobuf compiler plugin and MSBuild integration support for generating gRPC client/server C# code. Binaries are available for Windows, Linux and MacOS.</description>
+    <version>$version$</version>
+    <authors>Google Inc.</authors>
+    <owners>grpc-packages</owners>
+    <licenseUrl>https://github.com/grpc/grpc/blob/master/LICENSE</licenseUrl>
+    <projectUrl>https://github.com/grpc/grpc</projectUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <releaseNotes>Release $version$</releaseNotes>
+    <copyright>Copyright 2015, Google Inc.</copyright>
+    <tags>gRPC RPC Protocol HTTP/2</tags>
+	<developmentDependency>true</developmentDependency>
+  </metadata>
```

Comment:
I assume this package cannot depend on Grpc.Tools because that one is a solution level package while this one is project level package. is  that right?